oval_graph/client.py

```diff
@@ -31,20 +31,24 @@ def run_gui_and_return_answers(self):
             import inquirer
             return inquirer.prompt(self.get_questions())
         except ImportError:
-            print('== The Rule IDs ==')
-            rules = self.search_rules_id()
-            if self.show_fail_rules:
-                rules = self._get_only_fail_rule(rules)
-            for rule in rules:
-                print(rule['id_rule'] + r'\b')
-            if self.show_not_selected_rules:
-                print('== The not selected rule IDs ==')
-                for rule in self._get_wanted_not_selected_rules():
-                    print(rule['id_rule'] + '(Not selected)')
-            print("You haven't got installed inquirer lib. "
-                  "Please copy id rule with you want use and put it in command")
+            print(self.get_selection_rules())
             return None
 
+    def get_selection_rules(self):
+        out = '== The Rule IDs ==\n'
```

Comment:
When you create a long multi-line string, it is best to create a list of lines, and make the string from list of lines using `"\n".join(lol)` as the last step.

This method also mixes data presentation and data gathering, I recommend to split it - first gather list of rules, and transform the list to a string that should be displayed in another method.